packages/request-client.js/src/api/request.ts

```diff
@@ -133,9 +143,13 @@ export default class Request {
     // refresh the local request data
     const requestData = await this.refresh();
 
-    acceptResult.on('confirmed', async () => {
-      requestData.emit('confirmed', await this.refresh());
-    });
+    acceptResult
+      .on('confirmed', async () => {
+        requestData.emit('confirmed', await this.refresh());
+      })
+      .on('error', error => {
+        this.emitter.emit('error', error);
```

Comment:
Mhh why `confirmationErrorOccurred` is not set here ?

Reply:
It is only for the creation, I renamed the variable